Sliding-Images-Easy.md

```diff
@@ -0,0 +1,249 @@
+# Sliding Images Easy
+
+This tutorial will walk you through building an image slider using the [jQuery](https://jquery.com/) library.
+
+[![Screenshot showing Slider on CodePen](https://i.gyazo.com/234697e8dd7838d1f75f987c8fc68adc.png)](https://codepen.io/atjonathan/pen/BKMxxq)
```

Comment:
@atjonathan This is not what I meant.... :anguished: 

Add the optimized small size *gif* image ( try to make it < 500 KB) (resolution 800 px x 600 px) and then link it to the code pen like you've done here